### DISCUSSION TITLE
Shows unused labels in Camera Events "labels" dropdown menu

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#4709] [General]
FYI
When you select Events from the home page, then select a particular camera, the events dropdown lists events for all cameras and not just the one(s) specified for that camera. For example one of my cameras only has "bottle" defined, however if I select that camera, labels defined for all cameras are are displayed in the dropdown list.
It would be sleek/polished to only show labels defined for the chosen camera.

This does not bother me just thought I mention it.

PS: English is my second language so I'm not sure if "sleek/polished" are the right words to use above.

**Comment by NickM-27** [comment]
This is going to be better in the next version when all the filters are multi select

**Reply by 18923** [reply]
Super, I should have guessed you were on it.
Amazing work you are doing. 👍
